test/WebSites/MinimalAppWithConstructorDelay/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddEndpointsApiExplorer();
+
+var documentFilter = new DocumentFilter();
+var operationFilter = new OperationFilter();
+for (int i = 0; i < 1; i++)
+{
+    var x = i;
+    builder.Services.AddSwaggerGen(c =>
+    {
+        c.SwaggerDoc($"title-{x}", new OpenApiInfo { Title = $"SMARTR ({i})", Version = "1.0" });
```

Comment:
What's "SMARTR"?

Reply:
copy paste from reproduction code in the issue